examples/jsm/loaders/KTX2Loader.js

```diff
@@ -524,8 +524,8 @@ class KTX2Container {
 
 		for ( var level = 0; level < this.header.levelCount; level ++ ) {
 
-			var levelWidth = Math.ceil( width / Math.pow( 2, level ) );
-			var levelHeight = Math.ceil( height / Math.pow( 2, level ) );
+			var levelWidth = Math.floor( width / ( 1 << level ) ) || 1;
```

Comment:
Might as well use `>>` here? `(width >> level) || 1`

Reply:
(alternatively if readability is crucial it seems like Math.pow is a tiny bit nicer)